ex1/Tema3-V.4-Daniel Paraschiv.js

```diff
@@ -0,0 +1,38 @@
+document.addEventListener("DOMContentLoaded", () => {
+
```

Comment:
daca pui aici un `console.log("dom loaded")` de cate ori apare acest mesaj in browser?